File: Python/response/mms/RetrieveMMSResponse.py
from response.mms.InboundMessageList import InboundMessageList

class RetrieveMMSResponse:
	"""retrieve a list of messages sent to your Web application"""
	
	def __init__(self):
		"""Default class constructor"""
		self.httpResponseCode=0
		self.contentType=None
		self.location=None
		self.httpResponseCode=0
		self.contentType=None
		self.location=None
		self.inboundMessageList=None
	
	def getHTTPResponseCode(self):
		"""getter for httpResponseCode : the numeric status code result of the HTTP response"""
		return self.httpResponseCode
	
	def setHTTPResponseCode(self,httpResponseCode):
		"""setter for httpResponseCode : the numeric status code result of the HTTP response"""
		self.httpResponseCode=httpResponseCode
	
	def getContentType(self):
		"""getter for contentType : the resulting MIME content type of the HTTP response"""
		return self.contentType
	
	def setContentType(self,contentType):
		"""setter for contentType : the resulting MIME content type of the HTTP response"""
		self.contentType=contentType
	
	def getLocation(self):
		"""getter for location : the contents of the HTTP response location header"""
		return self.location
	
	def setLocation(self,location):
		"""setter for location : the contents of the HTTP response location header"""
		self.location=location
	
	def getInboundMessageList(self):
		"""getter for inboundMessageList : the value of the inboundMessageList field"""
		return self.inboundMessageList
	
	def setInboundMessageList(self,inboundMessageList):
		"""setter for inboundMessageList : the value of the inboundMessageList field"""
		self.inboundMessageList=inboundMessageList
	
	def setInboundMessageListJSON(self,jsondata):
		self.inboundMessageList=InboundMessageList(jsondata)
	

File: Ruby/response/mms/RetrieveMMSResponse.rb
require "response/mms/InboundMessageList"

class RetrieveMMSResponse

	def initialize
		@httpResponseCode=0
		@contentType=nil
		@location=nil
		@inboundMessageList=nil
	end

	def getHTTPResponseCode
		@httpResponseCode
	end

	def setHTTPResponseCode(httpResponseCode)
		@httpResponseCode=httpResponseCode
	end

	def getContentType
		@contentType
	end

	def setContentType(contentType)
		@contentType=contentType
	end

	def getLocation
		@location
	end

	def setLocation(location)
		@location=location
	end

	def getInboundMessageList
		@inboundMessageList
	end

	def setInboundMessageList(inboundMessageList)
		@inboundMessageList=inboundMessageList
	end

	def setInboundMessageListJSON(jsondata)
		@inboundMessageList=InboundMessageList.new
		@inboundMessageList.initializeJSON(jsondata)
	end

end 
